conf/mini_provisioner/v2/ha_setup.yaml

```diff
@@ -1,42 +1,53 @@
-# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
-#
-# This program is free software: you can redistribute it and/or modify it under the
-# terms of the GNU Affero General Public License as published by the Free Software
-# Foundation, either version 3 of the License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful, but WITHOUT ANY
-# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License along
-# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
-# about this software or licensing, please email opensource@seagate.com or
-# cortx-questions@seagate.com.
-
-ha:
-    post_install:
-        script: /opt/seagate/cortx/ha/conf/script/ha_setup.py
-        args: post_install
-
-    init:
-        script: /opt/seagate/cortx/ha/conf/script/ha_setup.py
-        args: init
-
-    config:
-        script: /opt/seagate/cortx/ha/conf/script/ha_setup.py
-        args: config
-
-    test:
-        script: /opt/seagate/cortx/ha/conf/script/ha_setup.py
-        args: test
-
-    reset:
-        script: /opt/seagate/cortx/ha/conf/script/ha_setup.py
-        args: reset
-
-    refresh_config:
-        script: /opt/seagate/cortx/ha/conf/script/ha_setup.py
-        args: replace_node
-
-support_bundle:
-    - cortxha support_bundle create
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+ha:
+    post_install:
+        cmd: ha_setup post_install

```

Comment:
Do we need ha_setup in cmd itself, I think post_install is fine. It creates extra confusion. Applies everywhere in the cmd.

Reply:
Yes, we need. This 'cmd' will be read by provisioner and then run.